### PR TITLE
Add new callback interface to receive the Capture creation event

### PR DESCRIPTION
**Pull request by satsukies**
We provide the callback interface to notify of finished create the Capture.
You can trigger any process when the Capture is successfully created.
e.g. logging application states, saving snapshot data, etc...

### CODE REVIEW

File: sdk/src/test/java/com/deploygate/sdk/DeployGateSdkConfigurationInterfaceTest.java [hunk 1]
```diff
@@ -46,6 +47,30 @@ public void onUpdateAvailable(int revision, String versionName, int versionCode)
...
+            public void onStatusChanged(boolean isManaged, boolean isAuthorized, String loginUsername, boolean isStopped) {
+                // no-op
+            }
+        };
+        updateAvailableCallback = new DeployGateUpdateAvailableCallback() {
+            @Override
+            public void onUpdateAvailable(int revision, String versionName, int versionCode) {
+                // no-op
+            }
+        };
```
Comment (satsukies) [inline sdk/src/test/java/com/deploygate/sdk/DeployGateSdkConfigurationInterfaceTest.java:67]: I forgot to initialize these variables.
ref. #95

**Review by jmatsu — CHANGES_REQUESTED** [review]
Please check the CI status.

**Review by jmatsu — APPROVED** [review]
LGTM

**Comment by satsukies** [comment]
Thank you!

**Pull request merged**